app/router/user.py

```diff
@@ -211,16 +211,25 @@ def get_user_jobs(
                 status_code=status.HTTP_409_CONFLICT, detail="Wrong filter"
             )
         if manage_tab == s.Job.TabFilter.PENDING:
-            log(log.INFO, "Pending tab, getting jobs ids for user: (%s)", str(current_user.id))
+            log(
+                log.INFO,
+                "Pending tab, getting jobs ids for user: (%s)",
+                str(current_user.id),

```

Comment:
when we want to format number we use next
``` python
%d
```

so final fix will look like this:
``` python
        log(
                log.INFO,
                "Pending tab, getting jobs ids for user: (%d)",
                current_user.id,
)
```